packages/@sanity/types/src/schema/types.ts

```diff
@@ -267,7 +246,7 @@ export interface ObjectSchemaType extends BaseSchemaType {
   // Note: `path` is a string in the _specification_, but converted to a
   // string array in the schema normalization/compilation step
   // eslint-disable-next-line camelcase
-  __experimental_search?: {path: string[]; weight: number; mapWith?: string}[]
```

Comment:
was `?` intended to be removed?

Reply:
It was not - thanks for catching that!

Reply:
ah, actually it was intentional. If no search config is specified the schema compiler will resolve one, so it will never be undefined.